src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -94,6 +96,11 @@ public Result applyFilter(final String line, final int entireLength) {
       return getFlutterDoctorResult(line, entireLength - line.length());
     }
 
+    // Check for "restart" action in debugging output.
+    if (line.startsWith("you may need to restart the app")) {
```

Comment:
You might consider putting this text in the bundle. It might be easier to find it when flutter changes the wording.